index.html

```diff
@@ -461,16 +467,89 @@ <h5 class="head-text-1">LET'S CONNECT</h5>
                 class="collapsible about-collapsible-button head-text-2"
               >
                 Skills
-                <object
-                  data="assets/icons/chevron.svg"
-                ></object>
+                <object data="assets/icons/chevron.svg"></object>
               </button>
             </div>
             <hr class="divider-1" />
           </li>
-
         </ul>
       </div>
     </section>
+
+    <!--CONTACT AREA-->
+    <section id="ContactForm" class="contact-section">
+      <div class="contact-background">
+        <img
+          class="illus-svg vector10"
+          src="assets/illustrations/vector-7.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector11"
+          src="assets/illustrations/vector-6.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg ellipse2"
+          src="assets/illustrations/ellipse-3.svg"
+          alt="###"
+        />
+      </div>
+
+      <div class="contact-stack">
+        <div class="contact-banner">
+          <h1 class="contact-header banner-content">Contact me</h1>
```

Comment:
- [ ] Kindly change this from `h1` to any other `hx` following your headers hierarchy. It's good practice to include one and only one `h1` element on the page which represents the page's main title.